torch/_inductor/codegen/wrapper.py

```diff
@@ -774,7 +775,22 @@ def generate_kernel_call(
                 stream_ptr = f"c_void_p({stream_name})"
                 self.writeline(f"{name}.{name}({call_args_str}, {stream_ptr})")
         else:
-            self.writeline(self.wrap_kernel_call(name, call_args))
+            if V.graph.aot_mode and config.aot_inductor.abi_compatible:
+                from .cpp import DTYPE_TO_CPP
+
+                new_args = []
+                for arg in call_args:
+                    var_name = f"var_{next(self.arg_var_id)}"
+                    self.writeline(f"void *{var_name}{self.ending}")
```

Comment:
Nit: we can declare this as `{cpp_dtype}*` to avoid the type conversion later.

Reply:
Hmm, if we declare ```var_name``` to be ```{cpp_dtype}*```, we will need a cast for being accepted by ```aoti_torch_get_data_ptr```, which takes ```void**```. I think I will just keep the current form.